Config.cmake.in

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
```

Comment:
As far as I understand `check_required_components` will not work without `@PACKAGE_INIT@`. Just copy/paste the `Config.cmake.in` from the link I provided.